test/test.hs

```diff
@@ -1233,17 +1650,17 @@ tests = testGroup "hevm"
               }
             }
             |]
-          let pre vm = (Lit 0) .== view (state . callvalue) vm
+          let pre vm = (Lit 0) .== Control.Lens.view (state . callvalue) vm
               post prestate poststate =
                 let (x,y) = case getStaticAbiArgs 2 prestate of
                         [x',y'] -> (x',y')
                         _ -> error "expected 2 args"
-                    this = Expr.litAddr $ view (state . codeContract) prestate
-                    prestore =  view (env . storage) prestate
+                    this = Expr.litAddr $ Control.Lens.view (state . codeContract) prestate
```

Comment:
Can we maybe make this a little more compact by fiddling with the import defintions?

Reply:
Fixed back to what it was originally :)